lib/services/resource.go

```diff
@@ -161,6 +161,8 @@ func ParseShortcut(in string) (string, error) {
 		return KindSemaphore, nil
 	case KindKubeService, "kube_services":
 		return KindKubeService, nil
+	case types.KindPAMConfig, "pam_configs":
```

Comment:
Hrm, PAM configuration can never be plural right?

Reply:
No, mostly added this because I think @fspmarshall mentioned it somewhere and thought something may be relying on it.

Reply:
Plural aliases are nice for being able to execute commands like `tctl get users`, which is more intuitive than `tctl get user`.  It isn't relied upon by anything programmatic, and probably isn't necessary here.